verification/verification_test.go

```diff
@@ -238,9 +238,9 @@ func testInitContext(d TestDPEInstance, t *testing.T) {
 func TestCertifyKey(t *testing.T) {
```

Comment:
Can you add a test case here to make sure that if CertifyKey is called with is_ca set to true/false, the right values get set in the cert?